domains/kappurumedia.json

```diff
@@ -6,6 +6,7 @@
     "email": "abramsatria7@gmail.com"
   },
   "record": {
+    "TXT": "dh=ce5c5f60ae66a7f508b44432fe2d9a0b51038fa8",
```

Comment:
Cannot have TXT and CNAME records together.

Reply:
how to add discord verif then

Reply:
add another json file?

Reply:
> add another json file?

Yes, it would be _discord.kappurumedia.json